runtime/src/execute_cost_table.rs

```diff
@@ -69,15 +123,23 @@ impl ExecuteCostTable {
                 .map(|(key, _)| key)
                 .expect("cannot find mode from cost table");
 
-            *self.table.get(key).unwrap()
+            self.table.get(key).unwrap().get_ema()
         }
     }
 
-    /// returns None if program doesn't exist in table. In this case,
+    /// return program average cost units, or None if program doesn't exist in table.
+    pub fn get_average_program_units(&self, key: &Pubkey) -> Option<u64> {
+        let aggregated_variance_stats = self.table.get(key)?;
+        Some(aggregated_variance_stats.get_ema())
+    }
+
+    /// returns inflated program cost units, which is `ema + 2*stddev`, or None
+    /// if program doesn't exist in table. In this case,
     /// `get_default_units()`, `get_average_units()` or `get_statistical_mode_units()`
     /// can be used to assign a value to new program.
-    pub fn get_cost(&self, key: &Pubkey) -> Option<&u64> {
-        self.table.get(key)
+    pub fn get_inflated_program_units(&self, key: &Pubkey) -> Option<u64> {
+        let aggregated_variance_stats = self.table.get(key)?;
+        Some(aggregated_variance_stats.get_ema() + 2 * aggregated_variance_stats.get_stddev())
```

Comment:
it'd be good to use checked math here, but we'd need to switch away from `Option` for the return to discern

Reply:
can address this in issue #23971